doc/man/8/radosgw.rst

```diff
@@ -137,50 +141,47 @@ Apache 2.4, needs to be configured for use with localhost tcp.
 
 		SetEnv proxy-nokeepalive 1
 
-		ProxyPass / fcgi://127.0.01:9000/
-		</VirtualHost>
-
-#. Modify Apache's configuration file so that ``mod_proxy_fcgi`` can be used
-   through unix domain socket.
-
-   Debian/Ubuntu::
-
-		<VirtualHost *:80>
-		ServerName localhost
-		DocumentRoot /var/www/html
-
-		ErrorLog /var/log/apache2/error.log
-		CustomLog /var/log/apache2/access.log combined
-
-		LogLevel debug
+		ProxyPass / fcgi://localhost:9000/
 
+		</VirtualHost>
 
-		RewriteEngine On
+#. For distros with Apache 2.4.9 or later that support Unix Domain Socket,
+   append the following configuration to ``/etc/ceph/ceph.conf``::
 
-		RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization},L]
+	[client.radosgw.gateway]
+	host = {hostname}
+	keyring = /etc/ceph/ceph.client.radosgw.keyring
+	rgw socket path = /var/run/ceph/ceph.radosgw.gateway.fastcgi.sock
+	log file = /var/log/radosgw/client.radosgw.gateway.log
+	rgw print continue = false
 
-		ProxyPass / unix:///tmp/.radosgw.sock|fcgi://localhost:9000/ disablereuse=On
-		</VirtualHost>
+#. Add the following content in ``/etc/httpd/conf.d/rgw.conf``::
 
    CentOS/RHEL::
 
-		<VirtualHost *:80>
+		<VirtualHost {IP ADDRESS}:80>
 		ServerName localhost
 		DocumentRoot /var/www/html
 
-		ErrorLog /var/log/httpd/error.log
-		CustomLog /var/log/httpd/access.log combined
-
-		LogLevel debug
+		ErrorLog /var/log/httpd/rgw_error.log
+		CustomLog /var/log/httpd/rgw_access.log combined
 
+		# LogLevel debug
 
 		RewriteEngine On
 
 		RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization},L]
 
-		ProxyPass / unix:///tmp/.radosgw.sock|fcgi://localhost:9000/ disablereuse=On
+		SetEnv proxy-nokeepalive 1
+
+		ProxyPass / unix:///var/run/ceph/ceph.radosgw.gateway.fastcgi.sock|fcgi://localhost:9000/
+
 		</VirtualHost>
 
+   The latest version of Ubuntu i.e, 14.04 has ``Apache 2.4.7`` which does not
+   have Unix Domain Support in it and as such has to be configured with localhost
+   tcp.
```

Comment:
Let's add a link to the Ubuntu bug report here. "This is tracked at https://bugs.launchpad.net/ubuntu/trusty/+source/apache2/+bug/1411030"